docs/architecture/components.md

```diff
@@ -0,0 +1,9 @@
+# Components
+
+## 1. Offline VC sharing component
+Transfer VC from Wallet to Verifier is facilitated by a react native library called [Tuvali](https://github.com/mosip/tuvali). This library is integrated into inji to allow offline VC transfer between mobile devices over Bluetooth Low Energy (BLE). 
+
+At present, offline VC transfer is supported between Android to Android and IOS to Android. VC transfer between Android to IOS is not supported yet.
```

Comment:
instead of words, how about a table representation in markdown? Words might be confusing?

Reply:
Represented in tabular form